### DISCUSSION TITLE
Swell.is + Builder.io integration

### DISCUSSION THREAD

**Discussion by Saadchr** [swellstores/community#420] [Help] [answered]
Hello,

I am new to swell.is environment.
What do you think of building an e-commerce using only Swell.IS + Builder.io?
I am planning to build websites for customers, will they handle their content management through Builder.io or through Swell.IS?
Is this a full no-code experience?

Have you exemples of websites using this stack?
Thanks

**Comment by ericingram** [comment, marked as answer]
I know a bunch of Swell users have integrated with Builder.io. In that case they would generally manage the frontend content in Builder  while maintaining store data in Swell.

Builder is a great solution. Maybe someone will chime in with their own experience.
